accepted/2021/preview-features/preview-features.md

```diff
@@ -0,0 +1,420 @@
+# Preview Features
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+Starting with .NET 5, we've moved to a fixed release schedule whereby we ship one
+version of .NET once a year, at RTM quality, with every other year being a long
+term support (LTS) release.
+
+This is in contrast to the .NET Framework days where we had multiple years
+for a given release, typically two to three years.
+
+Shipping more frequently has many advantages, such as being able to react
+quicker to trends and having an organization and customer base that is used to
+shipping & absorbing technology more quickly.
+
+With .NET Core, we've solved the problem of the extremely high in-place
+compatibility bar that .NET Framework has. This reduces risks for unintentional
+behavioral changes and also allows certain parts of .NET to make deliberate
+breaking changes if necessary.
+
+As a result, this makes it viable to innovate across runtime, libraries, and
+languages, allowing us to build more compelling features that would have been
+almost impossible to do on .NET Framework. Examples include static linking,
+default interface members, and ref structs (such as `Span<T>`).
```

Comment:
Also function pointers, covariant returns, etc ... Pick what you want here though, just wanted to call out other examples :smile: